webpack.config.js

```diff
@@ -58,7 +59,11 @@ const common = ({ platform }) => ({
       '@': path.join(__dirname, 'src'),
     },
   },
-  watchOptions
+  watchOptions,
+  plugins: [new Dotenv({
+    // Required to ensure Dotenv doesn't override any existing env vars that have been set by the system
+    systemvars: true
+  })],
```

Comment:
Adds `dotenv-webpack` to read in vars from `.env`

Reply:
Might just be misunderstanding this, but how would this work in production?

The project is built in teamcity, and built version is pushed to S3, which is downloaded and used by the server in AWS. We set all the environment variables for production as system environment variables on the server itself which we want to access at runtine. Whereas this plugin seems like it overwrites anything with `process.env` at build time.

Reply:
it should work the other way, any existing system variables should win out where there is a conflict.

Defo one to test on `code` though!

Reply:
I think from the understanding of the plugin, it replaces anything that is `process.env` at build time, so using the `systemvars` flag it would only pick up system environment variables set on the teamcity server. But yeah do test on `CODE`, and maybe check what's outputted in the bundle when running `yarn build` locally, since we still want `process.env` to be referenced in the bundle.

Reply:
From my testing, without the `systemvars` flag, Dotenv will crash the party and override any system variables already set. This wasn't what we want so I set it to true and now it's more gentle.

I'm assuming it does this on a case by case basis, and doesn't destroy the entire `process.env` object

Reply:
I deployed this branch to `code` and this page loads and works as expected. What do you think, does this give us the required level of confidence to move forward?